packages/output/src/common/output-channel.ts

```diff
@@ -105,12 +105,22 @@ export class OutputChannelManager implements Disposable {
     }
 }
 
+export enum OutputChannelSeverity {
+    Error = 1,
+    Warning = 2,
+    Info = 3
+}
+export interface OutputChannelLine {
+    text: string;
+    severity: OutputChannelSeverity;
```

Comment:
Just one remark; let's make this optional. 99% of the cases it will be `Info`, so we do not have to repeat it again. Thoughts? Or make `lines` property `Array<OutputChannelLine  | string>` type.

Reply:
I will make it optional but for clients there won't be any differennce as in the API is optional already.

Reply:
> there won't be any differennce as in the API is optional already

Thanks for the heads-up; yes I have noticed it.